embark-ui/.npmrc

```diff
@@ -0,0 +1,3 @@
+engine-strict = true
+package-lock = false
```

Comment:
Goodbye `npm-shrinkwrap.json` and `package-lock.json`. 👋

Reply:
Same as before, you probably don't need that file anymore

Reply:
Still needed, if it is removed in `embark/` or `embark/embark-ui`, then in either of those directories:
* `npm install` will run because `engine-strict` is what enforces looking at the `"engines"` setting in `package.json`. The `node_modules` tree will be rebuilt and `package-lock.json` will be written.
* `npm install [pkg]` (try with `npm install bluebird`) will run, the `node_modules` tree will be rebuilt, `package.json` will be modified with a caret range dependency, and `package-lock.json` will be written. When `.npmrc` is in place with `engine-strict = true` the command will still run, rebuild `node_modules` (!!!) and modify `package.json` 😞😞😞 but with an exact version instead of a caret range, and no `package-lock.json` will be written.